gatsby-config.js

```diff
@@ -120,8 +120,9 @@ module.exports = {
         // markdownSuffix: "_md",
         // modelField: "model",
         models: ["AdminReferences", "AdminSeo", "Form", "Page", "Redirect"],
+        plugins: ["gatsby-ample-pages"],
         projectRoot: path.join(__dirname)
-        // seoField: "seo"
+        // seoField: "seo",
```

Comment:
It can. `gatsby-config.js` had a lot of comments that I have now removed. Since we're following a convention with our custom plugins, and documenting their usage, I don't think we need so many comments in this file.